include/seqan3/alphabet/container/concatenated_sequences.hpp

```diff
@@ -1351,6 +1255,86 @@ class concatenated_sequences
         data_delimiters.push_back(data_delimiters.back() + std::ranges::size(value));
     }
 
+    /*!\brief Appends an empty element to the end of the container.
+     *
+     * This conceptionally adds an element to the container,
+     * i.e. the "outer container" grows by 1 and the new back() will be empty.
+     *
+     * No iterators are invalidated.
+     *
+     * ### Complexity
+     *
+     * Amortised constant. Wort-case linear in size().
+     *
+     * ### Exceptions
+     *
+     * Basic exception guarantee, i.e. guaranteed not to leak, but container my contain invalid data after exceptions is
+     * thrown.
+     *
+     * \experimentalapi{Experimental since version 3.1.}
+     */
+    void push_back()
+    {
+        data_delimiters.push_back(data_delimiters.back());
+    }
+
+    /*!\brief Appends the given element-of-element value to the end of the underlying container.
+     * \param value The value to append.
+     *
+     * This conceptionally performs a `push_back()` on the `back()` of this container,
+     * i.e. that last inner container grows by 1.
+     *
+     * If the new concat_size() is greater than concat_capacity() then all iterators and references (including the past-the-end
+     * iterator) are invalidated. Otherwise only the past-the-end iterator is invalidated.
+     *
+     * ### Complexity
+     *
+     * Amortised constant. Wort-case linear in concat_size().
+     *
+     * ### Exceptions
+     *
+     * Basic exception guarantee, i.e. guaranteed not to leak, but container my contain invalid data after exceptions is
+     * thrown.
+     *
+     * \experimentalapi{Experimental since version 3.1.}
+     */
+    void last_push_back(std::ranges::range_value_t<underlying_container_type> const value)
+    {
+        data_values.push_back(value);
+        ++data_delimiters.back();
+    }
+
+    /*!\brief Appends the given elements to the end of the underlying container (increases size of last element by n).
+     * \tparam rng_type The type of range to be inserted; must satisfy \ref is_compatible_with_value_type.
+     * \param value The value to append.
+     *
+     * This conceptionally performs an `insert()` on the `back()` of this container,
+     * i.e. the last inner container grows by value.size().
+     *
+     * If the new concat_size() is greater than concat_capacity() then all iterators and references (including the past-the-end
+     * iterator) are invalidated. Otherwise only the past-the-end iterator is invalidated.
```

Comment:
```suggestion
     * If the new concat_size() is greater than concat_capacity(), all iterators and references (including the 
     * past-the-end iterator) are invalidated. Otherwise, only the past-the-end iterator is invalidated.
```